### ISSUE TITLE
Exception java.lang.ClassNotFoundException

### ISSUE THREAD

**Issue by PStellmann** [oxygenxml/web-author-sample-plugins#1]
Since I only want to add a custom css file I tried to create a minimal plugin (only using WebappCSSResource) based on this one and got a java.lang.ClassNotFoundException.

So I tried to install this plugin and got the same error.

From the documentation it looks like  you HAVE TO implement at least two interfaces...

**Comment by mihai-coanda** [comment]
Hello Patrick,

These is no requirement for a plugin to use java code as long as it uses only not java extensions.
The _WebappCSSResource_ extension type was added in version _21.1.1_ of the WebAuthor so it can only be used on _21.1.1_ and future versions.
Regards,
Michael

**Comment by PStellmann** [comment]
We installed WebAuthor about 2 months ago: 

> oXygen XML Web Author: 21.1, build 2019052218/2019052201

**Comment by ctalau** [comment]
You are right. Your version is not that old, however we released version 21.1.1 of Web Author less than a week ago and it introduced a new extension point.
